### DISCUSSION TITLE
阿里云和腾讯云压力上不去

### DISCUSSION THREAD

**Discussion by erguidai** [juicedata/juicefs#1886] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/57423481/164972796-7adadd2d-8e14-4c4a-9655-982b9ef456c4.png)
![image](https://user-images.githubusercontent.com/57423481/164972804-b5c6d71c-390d-4b06-b912-a29f29fbe811.png)
在使用阿里云和腾讯去测试的时候总压不上去，怎么原因

**Comment by erguidai** [comment]
![image](https://user-images.githubusercontent.com/57423481/164972882-cdc4e264-5e24-4751-a5f9-079803c53297.png)
用本地资源时正常

**Comment by davies** [comment, marked as answer]
是在哪里运行的测试？如果不在云里面，会需要通过公网来读写数据，会受限于你的网络带宽，延迟也会非常高，性能无法得到保证的。

建议在云里面开一个跟对象存储在同一个区的虚拟机来跑压测。

**Reply by erguidai** [reply]
问题已解决，非常非常感谢
